horace_core/admin/check_horace_mex.m

```diff
@@ -21,72 +18,46 @@
 % $Revision: 1738 $    $Date: 2019-03-12 14:31:09 +0000 (Tue, 12 Mar 2019) $
```

Comment:
is removal of the `$Revision ` lines going in a follow-up PR in Herbert and Horace?

Reply:
Yep, this was already a big PR